tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -96,65 +95,53 @@ def render(
         timestamp: float,
         filepath,
         base_size: tuple[int, int],
-        pixelRatio: float,
-        isLoading=False,
+        pixel_ratio: float,
+        is_loading=False,
+        gradient=False,
+        update_on_ratio_change=False,
     ):
-        """Renders an entry/element thumbnail for the GUI."""
-        adj_size: int = 1
-        image = None
-        pixmap = None
-        final = None
+        """Internal renderer. Renders an entry/element thumbnail for the GUI."""
+        image: Image.Image = None
+        pixmap: QPixmap = None
+        final: Image.Image = None
         extension: str = None
-        broken_thumb = False
-        # adj_font_size = math.floor(12 * pixelRatio)
-        if ThumbRenderer.font_pixel_ratio != pixelRatio:
-            ThumbRenderer.font_pixel_ratio = pixelRatio
+        resampling_method = Image.Resampling.BILINEAR
```

Comment:
added this to make it easier to change the resampling method used

Reply:
Guessing you saw #174! Much appreciated! 😁

Reply:
yep haha